### DISCUSSION TITLE
Hassio site to site

### DISCUSSION THREAD

**Discussion by 111lisu** [hassio-addons/addon-wireguard#195] [General]
I would like to connect a hassio that does not have a public IP to my hassio so that a friend can connect via my server to his server via a  phone.

![Site-to-Site](http://server888056.nazwa.pl/Andrzejvpn2.png)

Right now, accessing my server via VPN (wireguard) is working fine. Just how to add to the configuration in hassio2 as client to hassio1 and firewall rules

**Comment by apbodrov** [comment]
Hi! I was able to set up such a configuration. Traffic goes back and forth from two networks.

We believe that there are no problems getting through from the client to the server network (it already works out of the box). In order to get into the client's home network from the server, you need to do the following:

1. add a friend's network to allowed_ips (0.0.0.0/0 does not work due to a docker container error) in wireguard addon config. If there was a default (empty) before, then you need not forget the subnet with the address, for example `- 172.27.66.0/24`
2. log in to the vpn server host machine (https://community.home-assistant.io/t/howto-how-to-access-the-home-assistant-os-host-itself-over-ssh/263352)
2.1 register a route to a friend's network `ip route add 192.168.x.0/24 via 172.30.33.0` (the last IP address of the wireguard container on the server)
2.2 allow the rule in the firewall `iptables -A FORWARD -i eth0 -j ACCEPT`
